draft-ietf-quic-invariants.md

```diff
@@ -316,6 +316,10 @@ connection IDs gives clients some assurance that the server received the packet
 and that the Version Negotiation packet was not generated by an off-path
 attacker.
 
+A server MUST NOT apply any restrictions to the length of the connection IDs
+when deciding whether to send a Version Negotiation Packet, even if it only
+supports QUIC versions that restrict the acceptable lengths of connection IDs.
+
```

Comment:
Personally, when I was looking for the rules of when or when not to send a VN packet, I went to the Invariants doc. And I didn't really find much. IMO, the invariants doc should have all the generic rules: which fields to validate, which lengths to ignore (CIDs) and which to enforce (UDP payloads). As we get more QUIC versions, implementers shouldn't have to go searching through every document out there (ideally).

Reply:
I agree with @nibanks here, and I like @martinthomson's suggestion in https://github.com/quicwg/base-drafts/pull/4187/files#r501426920. I'll use that here instead of in -transport.

Reply:
Done in 05cc5d9af4c53185585ae682b8d0f12673a2ac53.